<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiverInfoWidget</class>
 <widget class="QTabWidget" name="ReceiverInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>470</height>
   </rect>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::South</enum>
  </property>
  <widget class="QWidget" name="firstReceiverTab">
   <attribute name="title">
    <string>接收机基本信息</string>
   </attribute>
   <widget class="QGroupBox" name="infoContainer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>接收机信息</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>接收机名称：</string>
       </property>
       <property name="buddy">
        <cstring>receiverNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="receiverNameEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>接收机IP：    </string>
       </property>
       <property name="buddy">
        <cstring>receiverIPEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="receiverIPEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>接收机端口：</string>
       </property>
       <property name="buddy">
        <cstring>receiverPortEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="receiverPortEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>接收机位置</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>经度：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="receiverLongitudeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>纬度：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="receiverLatitudeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="satelliteDistributionContainer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel  {
  border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="secondReceiverTab">
   <attribute name="title">
    <string>接收机观测值信息</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="thirdReceiverTab">
   <attribute name="title">
    <string>接收机星历信息</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
